app/services/add_to_cart_service.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddToCartService < ApplicationService
```

Comment:
do not add `service` as a suffix. It's always better to play with namespace
something like
```ruby
module SessionCarts
  class AddMenu < ::ApplicationService
```